spec/commands/process_raw_event_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+RSpec.describe "ProcessRawEventCommand" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.